voila/app.py

```diff
@@ -671,6 +680,13 @@ def init_handlers(self) -> List:
                         "no_cache_paths": ["/"],
                     },
                 ),
+                (
+                    url_path_join(
+                        self.server_url, r"/voila/api/contents%s" % path_regex
```

Comment:
Curious: could exposing this endpoint allow someone access the content of other files on disk when they are not supposed to? For example when starting voila with `voila basics.ipynb` and trying to access another notebook via this endpoint?

Reply:
Indeed this endpoint is leaking the file structure (not the content) of the current working directory in the case of starting Voila with one notebook, I'll fix it.

Reply:
ok thanks for confirming :+1:

Reply:
Fixed by https://github.com/voila-dashboards/voila/pull/1374/commits/9c48666ca4da98b3a7aef0d50711b64fb0f27c79